src/components/sidebar/sidebar.css

```diff
@@ -44,8 +44,9 @@
 }
 
 .sidebar-title {
-  font-size: 12px;
+  font-size: 1.2em;
```

Comment:
note that 1.2em is 13.2px. But maybe that's what we want ?

Reply:
We do, yes.